pkg/api/v1/types.go

```diff
@@ -1312,6 +1312,15 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty" description:"disable pod scheduling on the node; see http://releases.k8s.io/HEAD/docs/admin/node.md#manual-node-administration"`
 }
 
+type KubeletPorts struct {
```

Comment:
That's a good question. I don't (I believe). If there's a reason why it should be included I can try to do this. From the top of my head I don't know if it'll be easy, as I'm not certain that Kubelet knows about KubeProxyPort.

If we want to make it more general, we probably need to think of better name.